event_sink_clickhouse/sinks/user_retire.py

```diff
@@ -0,0 +1,43 @@
+"""User retirement sink"""
+import requests
+
+from event_sink_clickhouse.serializers import UserRetirementSerializer
+from event_sink_clickhouse.sinks.base_sink import ModelBaseSink
+
+
+class UserRetirementSink(ModelBaseSink):  # pylint: disable=abstract-method
+    """
+    Sink for user retirement events
+    """
+
+    model = "auth_user"
+    unique_key = "id"
+    clickhouse_table_name = ["user_profile", "external_id"]
```

Comment:
It's sad that we don't have a more data driven way to deal with this list, but I think it's ok. Can you put a comment here reminding people that this list should exactly match `EVENT_SINK_CLICKHOUSE_PII_MODELS` in the Aspects plugin? We'll probably want one there, too, to try to keep these from falling out of sync.

Reply:
And transform the PII_MODELS, and MODELS, settings into OVERRIDE_SETTINGS. So those cannot be overriden

Reply:
I don't know if this is a desired capability, but it is possible to define other sinks in plugins, so people may want to override those settings. I'm not 100% sure about the hierarchy of settings here, but I think if there's a default setting in this app, and a setting in tutor-contrib-aspects the latter will override. Then if an operator has other needs their specific config will take precedence. I think that would be my preference.

Reply:
(rebuild worked.)